src/pages/Save.tsx

```diff
@@ -190,7 +190,7 @@ export const Save: FC = () => {
               </p>
             </div>
             <Links>
-              <LinkButton highlight href="https://governance.mstable.org/">
+              <LinkButton highlight href="https://staking.mstable.app/">
```

Comment:
@dimsome are you sure it's `.app`? I think the correct one is https://staking.mstable.org/

Reply:
Both work, if we change to https://staking.mstable.org/, then we also need to change it on the homepage.

I am ok with either

Reply:
@dimsome fair... I think I was having some DNS issues :(